src/test/java/com/nerodesk/om/aws/AwsDocsTest.java

```diff
@@ -49,4 +71,59 @@ public void conformsToEqualsHashCodeContract() {
             .suppress(Warning.TRANSIENT_FIELDS)
             .verify();
     }
+
+    /**
+     * AwsDocs can list all docs.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void listsDocs() throws IOException {
+        final Bucket bucket = this.mockBucket();
+        final Docs docs = new AwsDocs(bucket);
+        final List<String> expected = Lists.newArrayList(bucket.list(""));
+        MatcherAssert.assertThat(
+            docs.names(),
+            Matchers.equalTo(expected)
+        );
+    }
+
+    /**
+     * AwsDocs can find a doc.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void findsDoc() throws IOException {
```

Comment:
@longtimeago so maybe I should add some subfolders with files to the MkBucket backing and check if the names returned make sense ?